### PR TITLE
Add `arrow` style and change box config

### LINKED ISSUE #8: Add new option for vertical left and right
I would like to create a box which has arrows pointing to the center:

```
↘↓↓↓↓↓↓↙
→ Box  ←
↗↑↑↑↑↑↑↖
```

But this is not possible at the moment because there is just one vertical option. I suggest making it optional and add `verticalLeft` and `verticalRight`. No breaking change, just an add-on.

### PR DESCRIPTION
**Pull request by muuvmuuv**
Vertical and Horizontal now can have both different characters per side. This allows more unique boxes.

Open for interface naming. Hard to have a descriptive name for all cases. Maybe a base interface could make sense.

Closes #8

### CODE REVIEW

**Comment by muuvmuuv** [comment]
Next would be to update `boxen` to work with the new syntax: https://github.com/sindresorhus/boxen/blob/master/index.js#L181-L184

**Comment by sindresorhus** [comment]
Thinking more about it. I think it's better to just do a breaking change instead of introducing multiple types here. Just update the built-in boxes to use the new config.

File: boxes.json [hunk 1]
```diff
@@ -4,55 +4,79 @@
 		"topRight": "┐",
 		"bottomRight": "┘",
 		"bottomLeft": "└",
-		"vertical": "│",
-		"horizontal": "─"
+		"verticalLeft": "│",
```
Comment (sindresorhus) [inline boxes.json]: What do you think of just making these `left`, `right, `top`, `bottom`?

Comment (muuvmuuv) [reply boxes.json]: Good thought. Would make it more consistent with "bottomRight" and would fit better to the word box (left side etc.)

Comment (sindresorhus) [reply boxes.json]: Ok. Let's do that then.

File: readme.md [hunk 2]
```diff
@@ -18,12 +18,14 @@ const cliBoxes = require('cli-boxes');
...
-    vertical: '│',
-    horizontal: '─'
+	topLeft: '┌',
+	topRight: '┐',
+	bottomRight: '┘',
+	bottomLeft: '└',
+	verticalLeft: '│',
+	verticalRight: '│',
+	horizontalTop: '─',
+	horizontalBottom: '─'
```
Comment (sindresorhus) [inline readme.md]: Needs to be updated.

Comment (muuvmuuv) [reply readme.md]: Good catch. Thought I did find and replace on all files.

**Pull request merged**